browser/src/control/Control.UIManager.ts

```diff
@@ -529,6 +529,8 @@ class UIManager extends L.Control {
 			// setup quickfind panel
 			this.map.quickFindPanel = JSDialog.QuickFindPanel(this.map);
 			this.map.addControl(this.map.quickFindPanel);
+			// Initialize QuickFindPanel in core
+			app.socket.sendMessage('uno .uno:QuickFind');
```

Comment:
Best would be to put that initialization in the constructor of the quick find panel.
It hides the implementation detail inside reusable box (user of that class doesn't have to know and do any additional step).

Also all related code will be in the single place.